docker-credential-ecr-login.1

```diff
@@ -0,0 +1,91 @@
+.\" Copyright 2018 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+.\"
+.\" Licensed under the Apache License, Version 2.0 (the
+.\" "License"). You may not use this file except in compliance
+.\" with the License. A copy of the License is located at
+.\"
+.\"      http://aws.amazon.com/apache2.0/
+.\"
+.\" or in the "license" file accompanying this file. This file is
+.\" distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR
+.\" CONDITIONS OF ANY KIND, either express or implied. See the
+.\" License for the specific language governing permissions and
+.\" limitations under the License.
+.TH DOCKER-CREDENTIAL-ECR-LOGIN 1 2018-10-29 AMAZON-WEB-SERVICES AMAZON-ELASTIC-CONTAINER-REGISTRY
+.SH NAME
+docker\-credential\-ecr\-login \- Amazon ECR Credential Helper for Docker
+.SH SYNOPSYS
+.B docker\-credential\-ecr\-login
+COMMAND
+
+.B docker\-credential\-ecr\-login
+-v
+.SH DESCRIPTION
+The Amazon ECR Credential Helper for Docker is a credential helper for the
+.IR docker(1)
+command that makes it easier to use Amazon Elastic Container Registry.
+.SH USAGE
+Place the
+.B docker\-credential\-ecr\-login
+binary on your
+.IR PATH
+and set the contents of your
+.IR ~/.docker/config.json
+to be
+\fI{"credsStore":"ecr-login"}\fP.
+With Docker 1.13.0 or greater, you can configure Docker to use different
+credential helpers for different registries.  To use this credential helper for
+a specific ECR registry, create a
+.IR credsHelper
+section with the URI of your ECR registry:
+
+.nf
+{
+  "credHelpers": {
+    "aws_account_id.dkr.ecr.region.amazonaws.com":"ecr-login"
+  }
+}
+.fi
+
+Once installed, you may use \fIdocker pull\fP and \fIdocker push\fP with ECR
+repositories, without running \fIdocker login\fP.
+.SH AWS CREDENTIALS
+The credential helper reads AWS credentials from standard locations, including
+environment variables, the shared credentials file (\fI~/.aws/credentials\fP),
+EC2 instance profiles, and ECS task roles.
+
+To use credentials associated with a different named profile in the shared
```

Comment:
I'd like to see the other environment variables documented as well (specifically `AWS_ECR_DISABLE_CACHE`), but definitely not a blocker on a huge value add for this package!